addon-test-support/@percy/ember/index.js

```diff
@@ -0,0 +1,128 @@
+let isPercyRunning = true;
+// Capture fetch before it's mutated by Pretender
+let PercyFetch = window.fetch;
+
+function envInfo() {
+  let frameworkVersion = () => {
+    if (window.QUnit) {
+      return `qunit/${window.QUnit.version}`;
+    } else if (window.Mocha) {
+      // Doesn't look easy to grab the version while in the browser
+      return `mocha/unknown`;
+    }
+
+    return 'unknown';
+  };
+
+  return `ember/${window.Ember.VERSION}/${frameworkVersion()}`;
+}
+
+function clientInfo() {
+  return `@percy/ember/v2.0.0`;
```

Comment:
I'd really like to figure out a way to not hard code this. BUT, turns out its hard to import the package.json in an ember addon. So how much time should be sunk into giving that a go? 😬